webserver/server/app/models/pagination/slotBoundsPagination.sql

```diff
@@ -0,0 +1,25 @@
+/* @name slotBoundsPagination */
+WITH MIN_HASH AS
+	(SELECT COALESCE("Transaction".ID,
+
+										-1) AS MIN_TX_ID,
+			SLOT AS MIN_SLOT
+		FROM "Transaction"
+		JOIN "Block" ON "Block".ID = "Transaction".BLOCK_ID
+		WHERE SLOT <= :low!
+		ORDER BY "Block".ID DESC, "Transaction".ID DESC
+		LIMIT 1),
+	MAX_HASH AS
+	(SELECT SLOT AS MAX_SLOT,
+			COALESCE(MAX("Transaction".ID),
+
+				-2) AS MAX_TX_ID
+		FROM "Transaction"
+		JOIN "Block" ON "Transaction".BLOCK_ID = "Block".ID
+		WHERE SLOT <= :high!
+		GROUP BY "Block".ID
+		ORDER BY "Block".ID DESC
```

Comment:
do you not need to ORDER BY `"Transaction".ID` here as well?

Reply:
well... not really because there is an aggregation `Max("Transaction".id)`, although maybe both queries should be the same.